src/services/axiosInstance.ts

```diff
@@ -33,6 +33,10 @@ export const authorize = (url: string, method: string) => {
 const instance = axios.create({
   baseURL: import.meta.env.VITE_BASE_URL,
   timeout: 15000,
+  headers: {
+    "Access-Control-Allow-Origin": "*",
```

Comment:
it's ok for the internship project but be awared that in real project, we must determine the http header's security options by enviroment